src/main/java/com/targomo/client/api/quality/criterion/StatisticsReachabilityCriterionDefinition.java

```diff
@@ -16,6 +17,7 @@
 @ToString(callSuper = true)
 @Getter
 @JsonDeserialize(builder = StatisticsReachabilityCriterionDefinition.StatisticsReachabilityCriterionDefinitionBuilderImpl.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class StatisticsReachabilityCriterionDefinition extends RoutingBasedCriterionDefinition implements StatisticsCriterionDefinition{
     
```

Comment:
Can you also add a release note in the readme pls? As well as for the other branch you merged earlier

Reply:
done